keyboards/handwired/sick_pad/config.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CONFIG_H
+#define CONFIG_H
```

Comment:
```suggestion
#pragma once
```

Prefer `#pragma once` to these old-style include guards.